src/logger.ts

```diff
@@ -0,0 +1,62 @@
+import pino from 'pino';
+import pinoHttp from 'pino-http';
+
+import { isDevEnv } from './helpers';
+
+// API log levels
+const logLevels = {
+  verbose: 10,
+  debug: 20,
+  info: 30,
+  warn: 40,
+  error: 50,
+};
+
+// Common logging configuration
+const loggingConfiguration = {
+  base: undefined,
+  level: process.env.STACKS_API_LOG_LEVEL_ENV_VAR || 'info',
+  customLevels: logLevels,
+  useOnlyCustomLevels: true,
+  messageKey: 'message',
+  timestamp: () => `,"timestamp":"${new Date(Date.now()).toISOString()}"`,
+  formatters: {
+    level: (label: any) => {
+      return { level: label.toLowerCase() };
+    },
+  },
```

Comment:
You can simplify this as
```suggestion
  timestamp: pino.stdTimeFunctions.isoTime,
  formatters: {
    level: (label: string, number: number) => ({ level: label }),
  },
```

Reply:
Yep, `time` works correctly